### DISCUSSION TITLE
Project has disappeared after you force changed my account.

### DISCUSSION THREAD

**Discussion by hellokingdom** [vercel/community#6257] [Account Issue] [answered]
### Question

Hi

I have two projects hosted with Vercel on the hobby plan I was accessing them via this URL:

https://vercel.com/hellokingdom

About an hour ago I got a message regarding your personal projects migration and now my account is completely broken. I am getting permission errors when trying to change ay of my details. I can't upgrade to PRO as my user now lacks permissions to add payment details and one of my projects has totally disappeared. The other has been renamed and put into a new 'Team' called James Hyatt's Projects.

What is going on? Really confused as to what happened? I was preparing to switch one to a Pro account and that has now been lost and I seem unable to manage my Personal account correctly.

**Comment by amyegan** [comment]
Hey @hellokingdom. Can you share screenshots of the permission errors you're seeing? If you can walk me through the steps you tried, that would also be good to know. 🙏

**Reply by hellokingdom** [reply, marked as answer]
Hi @amyegan thank you for getting back to me. I've just logged in again and after a few hours I can now see the original project back in my personal projects. 

I have since added a new project with a pro account so I can now safely delete the original one.

You can file this as solved.

Thank you.
